internal/acceptance/mws_customer_managed_keys_test.go

```diff
@@ -16,3 +16,18 @@ func TestMwsAccCustomerManagedKeys(t *testing.T) {
 		}`,
 	})
 }
+
+func TestGcpaAccCustomerManagedKeysForStorage(t *testing.T) {
+	accountLevel(t, step{
+		Template: `provider "databricks" {
+				host     = "{env.DATABRICKS_HOST}"
+			}
+			resource "databricks_mws_customer_managed_keys" "this" {
```

Comment:
```suggestion
		Template: `resource "databricks_mws_customer_managed_keys" "this" {
```

provider doesn't have to be initialized